tests/CLI/modules/vlan_tests.py

```diff
@@ -93,6 +95,163 @@ def test_vlan_edit(self, click):
+    def test_vlan_edit_not_any(self, click):
+        result = self.run_command(['vlan', 'edit', '100'])
+        self.assertTrue(result.exit_code, 2)
+        self.assertEqual("At least one option is required", result.exception.message)
+
+    def test_vlan_create_not_data_pod(self):
+        _mock = self.set_mock('SoftLayer_Product_Package', 'getItems')
+        _mock.return_value = SoftLayer_Product_Package.getItemsVLAN
+        order_mock = self.set_mock('SoftLayer_Product_Order', 'placeOrder')
+        order_mock.return_value = SoftLayer_Product_Order.vlan_placeOrder
+
+        result = self.run_command(['vlan', 'create', '--name', 'my_vlan', '--datacenter', '', '--pod', ''])
+        self.assertIsInstance(result.exception, exceptions.CLIAbort)
+        self.assertIn("--datacenter or --pod is required to", result.exception.message)
+
+    def test_vlan_create_network_extra_route_exception(self):
+        _mock = self.set_mock('SoftLayer_Product_Package', 'getItems')
+        _mock.return_value = SoftLayer_Product_Package.getItemsVLAN
+        order_mock = self.set_mock('SoftLayer_Product_Order', 'placeOrder')
+        order_mock.return_value = SoftLayer_Product_Order.vlan_placeOrder
+        result = self.run_command(['vlan', 'create',
+                                   '--name', 'TEST001',
+                                   '--datacenter', 'dal09',
+                                   '--network', 'private',
+                                   '--pod', 'dal09.pod01'
+                                   ])
+        self.assertIsInstance(result.exception, exceptions.CLIAbort)
+        self.assertIn("Unable to find pod name", result.exception.message)
+
+    @mock.patch('SoftLayer.CLI.formatting.confirm')
+    def test_vlan_create_network_priv(self, confirm_mock):
+        confirm_mock.return_value = True
+        get_pods_mock = self.set_mock('SoftLayer_Network_Pod', 'getAllObjects')
+        get_pods_mock.return_value = [{
+            'name': 'ams03.pod01',
+            'frontendRouterId': 468136,
+            'backendRouterId': 468132}]
+        result = self.run_command(['vlan', 'create',
+                                   '--name', 'TEST001',
+                                   '--datacenter', 'ams03',
+                                   '--network', 'private',
+                                   '--pod', 'ams03.pod01'
+                                   ])
+
+        self.assert_called_with('SoftLayer_Network_Pod', 'getAllObjects')
+        self.assertIsInstance(result.exception, ex.SoftLayerError)
+
+    @mock.patch('SoftLayer.CLI.formatting.no_going_back')
+    def test_vlan_create_option(self, confirm_mock):
+        confirm_mock.return_value = True
+        datacenter_mock = self.set_mock('SoftLayer_Location_Datacenter', 'getDatacenters')
+        datacenter_mock.return_value = [{'id': 814994,
+                                         'name': 'ams03'
+                                         }]
```

Comment:
This could be 1 line

Reply:
Changed the mock return value to  1 liner